app/controllers/projects_controller.rb

```diff
@@ -20,7 +20,7 @@ def create
 
     if @project.save!
       flash[:success] = "Successfully created project #{project_name}"
-      redirect_to projects_path
+      redirect_to new_project_time_path(@project)
     else
```

Comment:
This is more of a design preference, but I would have preferred both a route that creates a project and then just returns to the index where you can assign times later, and then a route that creates a project then goes to new times path. That's not how the mockup has it but the mockup is still crude anyway. Thoughts?

Reply:
Yea that makes more sense.

Reply:
This is also causing our tests to fail, please either move this back or tailor the test to the new behavior.

Reply:
ya will move it back